cves/CVE-2015-6826.yml

```diff
@@ -90,14 +98,23 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 3197c0aa87a3b7190e17d49e6fbc7b554e4b3f0a
-  note: ''
-- commit: ''
-  note: ''
+  note: Original fix commit for v2.8
+- commit: 05684cee424a6e440c6757d224a748a2ffe87dde
+  note: Backport of fix to v2.7.2
+- commit: 620b3e680c388af7dd4a2ef2eb9544dc9cbdc092
+  note: Backport of fix to v2.6.4
+- commit: 8696762b9a3b66b7dac4bd7b8022a4ba0f72153a
+  note: Backport of fix to v2.5.8
+- commit: 24f1698758f0bd2ed5968cde35ce96ad58ba4c8d
+  note: Backport of fix to v2.4.11
 vccs:
-- commit: 9472d37d8e137df1e78e973b8b0e0d5607a799d5
```

Comment:
I agree with removing this one from the VCCs - Seems the code was changing quite a bit during this time!

Reply:
Yep, it ended up being fruitful to check up on archeogit's identifications.